fcrepo-serialization/src/main/java/org/fcrepo/camel/serialization/SerializationRouter.java

```diff
@@ -129,4 +135,17 @@ public void configure() throws Exception {
                 "{{serialization.binaries}}${headers[CamelSerializationPath]}")
             .to("exec:rm");
     }
+
+    private List<Predicate> getUriFilter() {
```

Comment:
How about just creating an issue for it now, sleep on it, and decide whether or not to go forward with that refactoring if the level of ambivalence changes.  I'm ambivalent too.  In the meantime, this PR looks great.  I wouldn't hold up merging for this.